golden_response/staging/get_stat_all/result.json

```diff
@@ -1,765 +1,765 @@
 {
-  "placeData": {
-    "country/USA": {
-      "statVarData": {
-        "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita": {
-          "sourceSeries": [
+  "placeData":  {
```

Comment:
any idea why this keeps happening?

Reply:
not sure. it seems to be get_stat_* related, so we could start there.